Dockerfile-prod

```diff
@@ -0,0 +1,34 @@
+#Based on the template from niployments (https://github.com/NIAEFEUP/niployments) for react applications
+
+# First, building the static files
+FROM node:10.16.2-alpine as build
```

Comment:
`10.16.3`? Also, it seems that the latest LTS is already `12.13.0`. However that might have breaking changes, we need to analyze it. Still, the latest version for the dubnium LTS (10.x) is `10.17.0`.

I believe that we should bump this version everywhere to at least latest dubnium and later on probably latest LTS, erbium (`12.13.0`).

Reply:
We could just rename to node:10-alpine just to make sure it stays up to date within v10

Reply:
`dubnium-alpine`? Idk, it feels strange to trust in automatic bumping. I know it will be inside the same major release, but still...

Reply:
Sure, I will edit the `engines` field in package.json as well

Reply:
👍